ci/configure-vm.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+set -eu
+
+function checks() {
+    if [[ $(id -u) -ne 0 ]]; then
+        echo "[-] This script needs to be run as root!"
+        exit 1
+    fi
+
+    if [[ $# -eq 0 ]]; then
+        echo "[-] User name of the runner user account required as first argument!"
+        exit 1
+    else
+        username=$1
+    fi
+    workspace=/home/$username/actions-runner
+
+    # shellcheck disable=SC1091
+    source /usr/lib/os-release
+    case "$ID" in
+        fedora) ;;
+        *)
+            echo "[-] $NAME has not been tested with this script!"
+            exit 1
+            ;;
+    esac
+
+    if [[ ! -f $workspace/svc.sh ]]; then
+        echo "[-] Runner must be configured already!"
+        echo '[-] Follow all "Download" steps and the first "Configure" step on the "Create self-hosted runner" page.'
+        exit 1
+    fi
+}
+
+function resize_rootfs() {
+    echo "[+] Resizing root filesystem if necessary"
+    dev_mapper=$(df -H | grep /dev/mapper/ | cut -d ' ' -f 1)
+    if [[ -n ${dev_mapper:-} ]]; then
+        # This might fail if the volume has already been resized, don't make it
+        # fatal to the script
+        lvextend -l +100%FREE "$dev_mapper" || true
+        xfs_growfs "$dev_mapper"
+    fi
+}
+
+function update_install_packages() {
```

Comment:
Would you mind an explicit check for the distro here?  I think that it would be easy to get this wrong when adding a new distro as there is no reference to easily search for.

Reply:
Should I duplicate the check in `checks()` or should I just add a note to `checks()` about how to add support for another distribution to this?